VM/common.py

```diff
@@ -0,0 +1,28 @@
+import csv
+from logging import Logger
+import os
+
+from qcore import qclogging
+
+script_dir = os.path.dirname(os.path.abspath(__file__))
+NZ_CENTRE_LINE = os.path.join(script_dir, "../SrfGen/NHM/res/centre.txt")
+NZ_LAND_OUTLINE = os.path.join(script_dir, "../SrfGen/NHM/res/rough_land.txt")
+
+
+def temp_paths(ptemp):
+    vm_working_dir = os.path.join(ptemp, "output")
+    nzvm_cfg = os.path.join(ptemp, "nzvm.cfg")
+    vm_params_path = os.path.join(ptemp, "vm_params.yaml")
+    return (vm_working_dir, nzvm_cfg, vm_params_path)
```

Comment:
This probably shouldn't be a common function.  vm_params.yaml shouldn't default to being in a temporary directory and output/nzvm.cfg isn't needed half the time this function is called

Reply:
agreed